### PR TITLE
fix: select, menu, Readme

### PR DESCRIPTION
**Pull request by innabelaya**
Closes 1275, 1272, 1270

### CODE REVIEW

File: common.blocks/menu/menu.en.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ A block is used to create different types of menu.
 
 `menu` block allows a user to manage state, behavior and appearance of the menu and its nested components – [menu-item](../menu-item/menu-item.en.md) blocks.
 
+The block supports key-pad control. If `menu` block is in [focuse](#focused) (`focused` modifier with `true` value), type a menu item name on your keyboard to switch to the menu item. Use `Space bar` or `Enter` buttons to select the menu item.
```
Comment (tadatuta) [inline common.blocks/menu/menu.en.md]: focuse -> focus

Comment (sipayRT) [inline common.blocks/menu/menu.en.md]: [focuse]

File: common.blocks/menu/menu.ru.md [hunk 2]
```diff
@@ -36,6 +36,8 @@
 
 Блок предоставляет возможность изменять внешний вид меню и управлять поведением вложенных независимых блоков – [пунктов меню](../menu-item/menu-item.ru.md).
 
+Реализация блока поддерживает клавиатурное утправление: если блок `menu` находится [в фокусе](#focused) (установлен модификатор `focused` в значении `true`), ввод с клавиатуры одного из названий пунктов меню автоматически переводит фокус на данный пункт. Выбор происходит нажатием на кнопку `Пробел` или `Ввод`.
```
Comment (sipayRT) [inline common.blocks/menu/menu.ru.md]: уТправление

Comment (tadatuta) [inline common.blocks/menu/menu.ru.md]: у**т**правление

File: common.blocks/select/select.ru.md [hunk 3]
```diff
@@ -23,16 +23,20 @@
...
 | <a href="#optionsmaxheight">optionsMaxHeight</a> | <code>Number</code> | Максимальная высота выпадающего списка. |
 | <a href="#id">id</a> | <code>String</code> | Уникальный идентификатор раскрывающегося списка. |
 | <a href="#tab">tabIndex</a> | <code>Number</code> | Последовательность перехода между контролами при нажатии на <code>Tab</code>. |
 
-
 ## Описание блока
 
 Блок визуально представлен кнопкой и выпадающим списком.
 
+Реализация блока поддерживает клавиатурное утправление, если блок `select` находится [в фокусе](#focused) (установлен модификатор `focused` в значении `true`):
```
Comment (tadatuta) [inline common.blocks/select/select.ru.md]: у**т**правление

File: common.blocks/select/select.en.md [hunk 4]
```diff
@@ -163,7 +167,7 @@ The modifier could be applied to:
 }
 ```
 
-* A separate menu item
+* Tha separate menu item
```
Comment (sipayRT) [inline common.blocks/select/select.en.md]: Th**a**

File: common.blocks/select/select.en.md [hunk 5]
```diff
@@ -209,13 +213,13 @@ The modifier provides a focus to the block.
 
 #### `theme` modifier
 
-Available value: `'islands'`.
+Acceptable value: `'islands'`.
 
 Use case: `BEMJSON`.
 
-The modifier provides a custom design to the block.
+The modifier gives the block a custom design.
```
Comment (sipayRT) [inline common.blocks/select/select.en.md:220]: maybe '**to** the block'?

File: common.blocks/select/select.en.md [hunk 6]
```diff
@@ -377,9 +381,9 @@ Specifies:
 
 Type: `String`.
 
-Specifies content of the button if any menu item is not selected.
+Specifies the button lable if any menu item is not selected.
 
-Use for the button with <a href="#mode-check">type modifier with check value</a> or with <a href="#mode-radiocheck">radio-check</a> value.
+Use for the button when the <a href="#mode-check">type modifier is set check</a> or to <a href="#mode-radiocheck">radio-check</a> value.
```
Comment (sipayRT) [inline common.blocks/select/select.en.md]: I propose:
Use for the button when the type modifier is set to <a href="#mode-check">check</a> or to <a href="#mode-radiocheck">radio-check</a> value.

File: common.blocks/select/select.ru.md (hunk 3 above)
Comment (sipayRT) [inline common.blocks/select/select.ru.md]: уТправление

File: common.blocks/select/select.en.md (hunk 6 above)
Comment (tadatuta) [inline common.blocks/select/select.en.md]: @innabelaya what do you think?

Comment (innabelaya) [inline common.blocks/select/select.en.md]: corrected: Use for the button when the `type` modifier is set to <a href="#mode-check">check</a> or to <a href="#mode-radiocheck">radio-check</a> value.

File: common.blocks/select/select.en.md (hunk 5 above)
Comment (innabelaya) [inline common.blocks/select/select.en.md:220]: The sentence is correct. The verb **give** (without to) has more suitable meaning.

**Pull request merged**